[2024-09-12T13:06:06.188-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-12T13:06:06.194-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_test.update_data manual__2024-09-12T16:06:02.631028+00:00 [queued]>
[2024-09-12T13:06:06.198-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_test.update_data manual__2024-09-12T16:06:02.631028+00:00 [queued]>
[2024-09-12T13:06:06.198-0300] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-12T13:06:06.211-0300] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): update_data> on 2024-09-12 16:06:02.631028+00:00
[2024-09-12T13:06:06.213-0300] {standard_task_runner.py:72} INFO - Started process 14625 to run task
[2024-09-12T13:06:06.215-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dag_test', 'update_data', 'manual__2024-09-12T16:06:02.631028+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmppr4_dg9l']
[2024-09-12T13:06:06.216-0300] {standard_task_runner.py:105} INFO - Job 43: Subtask update_data
[2024-09-12T13:06:06.239-0300] {task_command.py:467} INFO - Running <TaskInstance: dag_test.update_data manual__2024-09-12T16:06:02.631028+00:00 [running]> on host DESKTOP-PEJJLJO.
[2024-09-12T13:06:06.286-0300] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_test' AIRFLOW_CTX_TASK_ID='update_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-12T16:06:02.631028+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-12T16:06:02.631028+00:00'
[2024-09-12T13:06:06.287-0300] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-12T13:06:06.299-0300] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-09-12T13:06:06.300-0300] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2024-09-12T13:06:07.292-0300] {dag_test.py:78} INFO - {'spreadsheetId': '1EGfO6zUlbZqMfXmMU7_CXh4PpYpQTFgCjyVuqM1fT3I', 'updatedRange': "'PÃ¡gina1'!A1:C2", 'updatedRows': 2, 'updatedColumns': 3, 'updatedCells': 6}
[2024-09-12T13:06:07.292-0300] {python.py:240} INFO - Done. Returned value was: None
[2024-09-12T13:06:07.295-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-12T13:06:07.296-0300] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_test, task_id=update_data, run_id=manual__2024-09-12T16:06:02.631028+00:00, execution_date=20240912T160602, start_date=20240912T160606, end_date=20240912T160607
[2024-09-12T13:06:07.352-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-12T13:06:07.363-0300] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-12T13:06:07.368-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
